docs/1.0. Certificate Provisioning.md

```diff
@@ -208,9 +212,9 @@ An EST Client MUST provide a method to replace the manufacturer-issued TLS Clien
 
 An EST Client MAY ignore all timestamps in the certificate validity periods during the certificate provisioning process if it does not accurately know the current time, see [Bootstrapping Remote Secure Key Infrastructures - Section 2.6.1](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-2.6.1) for more information.
 
```

Comment:
Suggest that this passage from the "Notes for Implementers" section is either:
- better placed here
- better placed  in the implementation guide
- better deleted
```suggestion
An EST Clients MUST have access to a good source of randomness in an acceptable amount of time. For more information on achieving this, see [BRSKI - Section 5.9.4](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-5.9.4) and [RFC 4086 - Randomness Requirements for Security][RFC-4086].
```